docs/source/ddp_comm_hooks.rst

```diff
@@ -82,6 +82,18 @@ PowerSGD Hooks
 .. autofunction:: powerSGD_hook
 .. autofunction:: batched_powerSGD_hook
 
+Debugging Communication Hooks
+-----------------------------
+
+As the name implies, debugging communication hooks are **only** used for debugging purpose.
+
+.. currentmodule:: torch.distributed.algorithms.ddp_comm_hooks.debugging_hooks
+
+.. warning ::
+    Debugging communication hooks do not necessarily output the correct results.
```

Comment:
Maybe expand on this a bit, and say that debugging comm. hooks are mostly meant for perf optimization tests, and thus shouldn't be used for accuracy purposes?

Reply:
I think the purpose has been emphasized in both the first sentence of this section as well as the docstring.